### PR TITLE
support db2 in springboot

### PR DESCRIPTION
**Pull request by doctormacky**
This patch is about support db2 for casbin policy.

### CODE REVIEW

File: src/main/java/org/casbin/adapter/DB2Adapter.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	 * @param jdbcTemplate
+	 * @param autoCreateTable
+	 */
+    public DB2Adapter(JdbcTemplate jdbcTemplate, boolean autoCreateTable) {
+        super(jdbcTemplate, autoCreateTable);
+    }
+
+    @Override
+    protected void initTable() {
+	    try {
```
Comment (fangzhengjin) [inline src/main/java/org/casbin/adapter/DB2Adapter.java]: tab麻烦换成空格

File: src/main/java/org/casbin/adapter/DB2Adapter.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            "  V2 VARCHAR(255) DEFAULT NULL ," +
+            "  V3 VARCHAR(255) DEFAULT NULL ," +
+            "  V4 VARCHAR(255) DEFAULT NULL ," +
+            "  V5 VARCHAR(255) DEFAULT NULL" +
+            ")";
+
+	/**
+	 * @param jdbcTemplate
+	 * @param autoCreateTable
+	 */
```
Comment (fangzhengjin) [inline src/main/java/org/casbin/adapter/DB2Adapter.java]: tab麻烦换成空格

**Review by fangzhengjin — CHANGES_REQUESTED** [review]
tab麻烦换成空格

**Comment by doctormacky** [comment]
close it as still have some format issue

**Pull request closed without merging**